7/apache/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN set -ex; \
 	; \
 	\
 	docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr; \
-	docker-php-ext-install -j "$(nproc)" \
+	docker-php-ext-install \
```

Comment:
Why are we dropping all the `-j` values?

Reply:
For consistency with the `wordpress` image. Is there a reason why this defaults to `1`?
https://github.com/docker-library/php/blob/04e67dc6bb76783d2d1b64513b8f42961e578645/docker-php-ext-install#L47

Reply:
Ah, then maybe we should update WordPress.

It defaults to `make`'s default (we don't explicitly `-j1` anywhere), mostly to be considerate of `php` users' machines -- for the official images though, we want to make full use of our build systems as much as possible (especially chasing the dragon of potentially faster builds), which is why we typically use `-j "$(nproc)"` in ours.

Reply:
I reverted this. I also noted that freetype support was missing in the Debian variant.